manifests/server/config.pp

```diff
@@ -35,6 +35,13 @@
     mode   => '0700',
   }
 
+  file { $mcollective::ssldir:
+    ensure => 'directory',
+    owner  => 'root',
+    group  => '0',
+    mode   => '0755',
+  }
```

Comment:
this might still fail in old puppet versions where all resources in this class could be parallelised.